### PR TITLE
Release 1.7.1

### PR DESCRIPTION
**Pull request by Kircheneer**

### CODE REVIEW

File: diffsync/store/__init__.py [hunk 1]
```diff
@@ -159,6 +159,24 @@ def get_or_instantiate(
 
         return obj, created
 
+    def get_or_add_model_instance(self, obj: "DiffSyncModel") -> Tuple["DiffSyncModel", bool]:
```
Comment (glennmatthews) [inline diffsync/store/__init__.py:162]: Given new API features, perhaps bump to 1.8.0 instead of 1.7.1?

**Review by glennmatthews — COMMENTED** [review]
Code changes look solid. Planning to update CHANGELOG.md?

**Comment by Kircheneer** [comment]
Yep, I will pull in #220 and make it 1.8.

**Comment by glennmatthews** [comment]
Closing in favor of #226.

**Pull request closed without merging**